src/CustomersList/Header/style.css

```diff
@@ -0,0 +1,36 @@
+body{
```

Comment:
I told you in previous comments, body styles shouldn't be here. There will be only one body element for whole react application and you can style body for every page you're designing like this. It should be in higher place like in entry mount file.